audit/projects/kubernetes-public/buckets/k8s-project-triage/metadata.txt

```diff
@@ -0,0 +1,18 @@
+gs://k8s-project-triage/ :
```

Comment:
Added by https://github.com/kubernetes/k8s.io/pull/2454

Reply:
This on me to manually remove, which I will do when https://github.com/kubernetes/k8s.io/pull/2461 merges